distributed/diagnostics/plugin.py

```diff
@@ -127,7 +127,8 @@ class WorkerPlugin:
     ...                 exc_info=exc_info
     ...             )
 
-    >>> plugin = ErrorLogger()
+    >>> import logger
```

Comment:
Hi @crendoncoiled thanks for the PR, I'm just trying to run the example but `import logger` doesn't work. I believe this is `import loggin` so maybe we can re-write this as. 

```suggestion
    >>> import logging as logger
```

Reply:
I see the typo! Fixed